core/src/main/scala/org/scalawag/jibe/backend/bash/BashCommander.scala

```diff
@@ -0,0 +1,33 @@
+package org.scalawag.jibe.backend.bash
+
+import org.scalawag.jibe.mandate.command.CommandArgument._
+
+object BashCommander {
+  def bashify(structure: StructureValue): Iterable[String] = {
```

Comment:
This is the method that takes the "generic" structure produced by the macro and turns it into bash-specific variable assignments.